### PR TITLE
Correct BytesReadUFS and BytesReadRemote after adding fuse in worker

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?
Fix the incorrect ufs read metrics by https://github.com/Alluxio/alluxio/pull/13767
Record the correct ufs read data.

BytesReadRemote -- represent all the bytes served by this worker to clients via gRPC, no matter the data is from worker storage or UFS
BytesReadDirect -- represent all the bytes served by this worker to worker internal client via worker internal call, no matter the data is from worker storage or UFS
BytesReadUfs -- represent all the data read from UFS, no matter it will be served to remote clients via gRPC or worker internal clients

### Why are the changes needed?
Otherwise, if users read from UFS through worker internal call, the ufs metrics will not be increased and show a wrong number.

### Does this PR introduce any user facing changes?
The BytesReadUfs and BytesReadUfsThroughput metrics will show correct data

### CODE REVIEW

**Comment by LuQQiu** [comment]
Now the ufs read metrics will be correct (no matter users read via gRPC or external call).

BytesReadDirect  = BytesReadDirect from worker storage + BytesReadDirect from UFS (BytesReadUFS)
BytesReadRemote = bytes read from this worker but not include ufs read.

I kind of feeling those metrics are messy....

**Comment by LuQQiu** [comment]
I am not that familiar with domain socket, but ideally
BytesReadRemote -- represent all the bytes served by this worker to clients via gRPC, no matter the data is from worker storage or UFS
BytesReadDirect -- represent all the bytes served by this worker to worker internal client via worker internal call, no matter the data is from worker storage or UFS
BytesReadUfs -- represent all the data read from UFS, no matter it will be served to remote clients via gRPC or worker internal clients

**Review by yuzhu — APPROVED** [review]
LGTM, thanks for the fix @LuQQiu 
I agree with your explanation of the metrics and we probably should include those definitions in our documentation, including how cache hit and miss rate should be calculated.

File: core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java [hunk 1]
```diff
@@ -90,6 +92,9 @@
    */
   private long mInStreamPos;
 
+  private Counter mCounter;
+  private Meter mMeter;
```
Comment (apc999) [inline core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java:96]: possibly `final`?

File: core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java [hunk 2]
```diff
@@ -134,6 +139,14 @@ private UnderFileSystemBlockReader(UnderFileSystemBlockMeta blockMeta, boolean p
     mUfsResource = ufsClient.acquireUfsResource();
     mUfsMountPointUri = ufsClient.getUfsMountPointUri();
     mIsPositionShort = positionShort;
+
```
Comment (apc999) [inline core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java:142]: nit: unnecessary empty line

**Review by apc999 — APPROVED** [review]
minor comments otherwise LGTM

**Comment by LuQQiu** [comment]
@yuzhu @apc999 Addressed the PR and modified the metrics to correctly show BytesReadUfs, BytesReadLocal and BytesReadRemote metrics

**Comment by LuQQiu** [comment]
alluxio
alluxio-bot, merge this please

**Review by apc999 — APPROVED** [review]
failed tests are flaky and fixed in master branch

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**